tutorials/Binary_Contingency_Scores.ipynb

```diff
@@ -0,0 +1,2738 @@
+    "# If you don't wish to visualise the binary \"event tables\", you can ignore this section.\n",
+    "# However, if you wish to visualise or utilise the the \"event tables\" directly, you can do so as follows:\n",
+    "\n",
+    "forecast_binary, observed_binary = event_operator.make_event_tables(simple_forecast, simple_obs)\n",
+    "print(forecast_binary)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dec28e21-6d2c-486c-8387-b6d164dce570",
+   "metadata": {},
+   "source": [
+    "## Step Two: Making Binary Contingency Information\n",
+    "\n",
+    "Once the event tables have been made, the next step is to make a contingency manager object. It is possible (and quicker) to make a contingency manager from the event operator directly, but we will first show the process as separate steps, and then show how to do the two things at once. The contingency manager stores the binary forecast event data, the binary observed event data and the contingency table itself, plus provides functions for aggregating the data along various dimensions, and calculating metrics from that information."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "1cab2d73-ca4b-4c90-b0e2-db940c0cd198",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Contingency Manager (xarray view of table):\n",
+      "<xarray.DataArray (contingency: 5)> Size: 40B\n",
+      "array([ 9,  6,  2,  1, 18])\n",
+      "Coordinates:\n",
+      "  * contingency  (contingency) <U11 220B 'tp_count' 'tn_count' ... 'total_count'\n"
+     ]
+    }
+   ],
+   "source": [
+    "contingency_manager = scores.categorical.BinaryContingencyManager(forecast_binary, observed_binary)\n",
+    "print(contingency_manager)  # Print a view of the contingency table"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "51d3d4d9-e890-4266-bd8f-c0ae44d147a2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Uncomment this line to view the full functionality provided by a contingency manager\n",
+    "# help(contingency_manager)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "17f971df-7d39-4ddd-9a9b-98b88e9a71dc",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Contingency Manager (xarray view of table):\n",
+      "<xarray.DataArray (contingency: 5)> Size: 40B\n",
+      "array([ 9,  6,  2,  1, 18])\n",
+      "Coordinates:\n",
+      "  * contingency  (contingency) <U11 220B 'tp_count' 'tn_count' ... 'total_count'\n"
+     ]
+    }
+   ],
+   "source": [
+    "# It is also possible, and briefer, to make the binary contingency table directly from the event operator, as follows:\n",
+    "\n",
+    "contingency_manager = event_operator.make_table(simple_forecast, simple_obs)\n",
+    "print(contingency_manager)"
```

Comment:
Again, this is clearer if you just use `.get_table()` rather than print

Reply:
done